<?xml version="1.0"?>
<launch>
  <rosparam param="pegasus_mavros_namespaces">
    [
      "uav0",
      "uav1",
      "uav2"
    ]
  </rosparam>
  <rosparam param="pegasus_local_transforms">
    [
      [ "map_0", "base_link_0"],
      [ "map_1", "base_link_1"],
      [ "map_2", "base_link_2"]
    ]
  </rosparam>
  <param name="map_origin_topic" value="/local_xy_origin"/>
  <param name="magnetic_declination" value="-3.2"/>
  <node pkg="pegasus_ros" type="pegasus_ros_map_util" name="pegasus_ros_map_util" respawn="false" output="screen">
  </node>
</launch>
